set-target.sh

```diff
@@ -28,6 +28,7 @@ trap "$(printf 'rm -rf "%q"' "$TMP")" EXIT
 (
     cd "$TMP"
     cp "$MODFILE" "go.mod"
+    go mod download bazil.org/fuse
     go list -f '-require={{.Path}}@{{.Version}}{{if .Replace}} -replace={{.Path}}@{{.Version}}={{.Replace}}{{end}}' -m all | tail -n+2  > args
```

Comment:
My understanding is that nothing special is happening with fuse here. Instead what's happening is that we're copying a go.mod file into a temp directory all by itself and trying to run `go list` on it. However, `go list` seems to refuse to run without access to a go.sum, since as of https://golang.org/doc/go1.16#go-command the default is that, for safety I assume, all tools assume the existence of the go.sum file and run effectively with `-mod=readonly`.

We can fix this either by copying over the go.sum file or by running `go list -mod=mod {rest of command}`. IDK if there's a good way to grab the .sum file, so just adding `-mod=mod` seems like the way to go.


--- 

Note: the reason `go mod download bazil.org/fuse` was showing up first is because it's the first entry in the go.mod file, and the reason this change was working is because it was triggering the go.sum file to be built.

Reply:
Fixed.